src/main/io/ledstrip.c

```diff
@@ -1048,6 +1083,50 @@ bool parseColor(uint8_t index, const char *colorConfig)
     return ok;
 }
 
+/*
+ * Redefine a color in a mode.
+ * */
+bool setModeColor(uint8_t modeIndex, uint8_t modeColorIndex, uint8_t colorIndex)
+{
+    bool ok = true;
```

Comment:
Why not just `return true;` / `return false;` ? `bool ok` is usually used to handle errors (setting it to false when something fails)., this is very confusing